src/content/reference/react/useId.md

```diff
@@ -30,35 +29,35 @@ function PasswordField() {
   // ...
 ```
 
-[See more examples below.](#usage)
+[Lihat lebih banyak contoh di bawah ini.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-`useId` does not take any parameters.
+`useId` tidak mengambil parameter apapun.
 
 #### Returns {/*returns*/}
```

Comment:
It can be translated accord to [this](https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769320)

"Mengembalikan" cc @resir014 @zainfathoni

Reply:
```suggestion
#### Kembalian {/*returns*/}
```

Reply:
updated